cmd/ping.go

```diff
@@ -71,5 +75,8 @@ func (o *pingOptions) run(cmd *cobra.Command) error {
 		fmt.Sprintf("Status: %s ", "Connected")
 
 	cmd.Println(response)
+	res, _ := p.MarshalBinary()
+	log.Debug("Server response:\n", string(res))
+
```

Comment:
I think we do not need this, given that the response of each request is already printed here:
https://github.com/reanahub/reana-client-go/blob/24440c20747488bc06118872d3c56b0b42544110/client/client.go#L49-L50